runtime/oti/vmconstantpool.xml

```diff
@@ -143,6 +143,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 	<classref name="jdk/internal/loader/NativeLibraries" versions="15-"/>
 	<classref name="jdk/internal/loader/NativeLibraries$NativeLibraryImpl" versions="15-"/>
+	
+	<classref name="java/lang/Record" versions="16-"/>
```

Comment:
Is this used anywhere now?

Reply:
Removed as `reflecthelp.c` remains unchanged.